.env.local

```
@@ -0,0 +1,48 @@
+# Environment Configuration
+
+# Application Environment
+NODE_ENV=development
+
+# Database Configuration
+DATABASE_URL=postgresql://postgres:Buckalew2025@localhost:5432/Buckalew?schema=public
+
+# Security Secrets
+NEXTAUTH_SECRET=bb121d85c1a58625ff783c0a4799fe0600f2913398c4d95c5582640a0ae85c9b
+ENCRYPTION_SECRET_KEY=6349bc01304a57e42c8b5e28003bac6c257fa24325f5bb61898bc455827bbeecaf2e613f9976e8c97ba2f481fbff27a9050e0d65dedd6177d8d5ece838c0905f
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The ENCRYPTION_SECRET_KEY key should go before the NEXTAUTH_SECRET key

(UnorderedKey)